build.gradle

```diff
@@ -285,7 +285,13 @@ dependencies {
     compile "com.icthh.xm.commons:xm-commons-lep:${xm_commons_version}"
     compile "com.icthh.xm.commons:xm-commons-timeline:${xm_commons_version}"
     compile "com.icthh.xm.commons:xm-commons-metric:${xm_commons_version}"
- 
+
+    compile "com.lowagie:itext:2.1.7"
+    compile ("net.sf.jasperreports:jasperreports:${jasperreports_version}") {
+        exclude group: 'com.lowagie', module: 'itext'
+    }
+    compile files('lib/timesnewroman-cyrillic-font.jar')
```

Comment:
There is a dependency in maven repo: https://mvnrepository.com/artifact/net.sf.jasperreports/jasperreports-fonts

Reply:
In case it is related only to Cyrillic fonts propose to remove direct dependency on them from codebase and add JAR to docker for particular environment